operator/pkg/central/values/translation/image_overrides.go

```diff
@@ -4,12 +4,11 @@ import "github.com/stackrox/rox/operator/pkg/images"
 
 var (
 	imageOverrides = images.Overrides{
-		images.Main:             "central.image.fullRef",
-		images.CentralDB:        "central.db.image.fullRef",
-		images.Scanner:          "scanner.image.fullRef",
-		images.ScannerDB:        "scanner.dbImage.fullRef",
-		images.ScannerV4DB:      "scannerV4.db.image.fullRef",
-		images.ScannerV4Indexer: "scannerV4.indexer.image.fullRef",
-		images.ScannerV4Matcher: "scannerV4.matcher.image.fullRef",
+		images.Main:        "central.image.fullRef",
+		images.CentralDB:   "central.db.image.fullRef",
+		images.Scanner:     "scanner.image.fullRef",
+		images.ScannerDB:   "scanner.dbImage.fullRef",
+		images.ScannerV4DB: "scannerV4.db.image.fullRef",
+		images.ScannerV4:   "scannerV4.image.fullRef",
```

Comment:
nit: reorder for consistency

Reply:
sorry, have only seen that now. I suggest to proceed.